docs/hub/models-uploading.md

```diff
@@ -66,10 +69,57 @@ Any repository that contains TensorBoard traces (filenames that contain `tfevent
 
 Models trained with 🤗 Transformers will generate [TensorBoard traces](https://huggingface.co/docs/transformers/main_classes/callback#transformers.integrations.TensorBoardCallback) by default if [`tensorboard`](https://pypi.org/project/tensorboard/) is installed.
 
-## Using Git
+## Using `huggingface_hub`
+
+In case your model is a (custom) PyTorch model, you can leverage the `PyTorchModelHubMixin` [class](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) available in the [huggingface_hub](https://github.com/huggingface/huggingface_hub) Python library. It is a minimal class which adds `from_pretrained` and `push_to_hub` capabilities to any `nn.Module`, along with download metrics.
```

Comment:
IMO, we should also link to https://huggingface.co/docs/huggingface_hub/guides/upload and explain the difference between the mixin approach v.s. simple upload approach. The `PyTorchModelHubMixin` can introduce some complexities in terms of integration compared to a simple upload with `HfApi`.